openspec/changes/safetensors-model-loader-v1/design.md

```
@@ -0,0 +1,305 @@
+## Context
+
+SmallAIOS today is an ONNX-only inference runtime. The full pipeline is:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)